[17.0] [MIG] automation_oca: Migration to 17.0

mjavint opened the pull request:
Superseed https://github.com/OCA/automation/pull/10 since there is no answer

etobella committed:
[ADD] automation_oca

luisDIXMIT committed:
[IMP] automation_oca: README.md

Commit:
[UPD] Update automation_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (218 of 218 strings)

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (218 of 218 strings)

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/it/

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 96.7% (211 of 218 strings)

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/es/

gregory-moka committed:
Added translation using Weblate (French)

gregory-moka committed:
Translated using Weblate (French)

Currently translated at 61.9% (135 of 218 strings)

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/fr/

etobella committed:
[FIX] automation_oca: Improve selection string on ranges

gaelTorrecillas committed:
[IMP] automation_oca: Allow some expressions in domain

This commit adds the corresponding imports and preparation
code for using some variables and expressions like `today()`,
`datetime`, `user`, etc in the domain (filter) of the automation
configuration.

gaelTorrecillas committed:
[16.0] automation_oca : add demo and help for new domain options

Commit:
[UPD] Update automation_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (220 of 220 strings)

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/it/

gaelTorrecillas committed:
[IMP] automation_oca: broadcast the domain in steps

[FIX] automation_oca: pass eval_context in step domain safe_eval

OCA-git-bot committed:
[BOT] post-merge updates

gaelTorrecillas committed:
[FIX] automation_oca: pass eval_context in record step domain safe_eval

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update automation_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (220 of 220 strings)

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/it/

christian-ramos-tecnativa commented:
Thanks for this, it seems like the link is wrong builded

christian-ramos-tecnativa commented:
Also, do you mind squashing the commit 92764e287b9d7018244d05a1cd4557fbf33d5c4b into 7d31035b98d6e6d0f94a93d5b9afc4666cc56016? That way will be much easier to review the changes done during the migration, right now there are a lot of HEAD tags on the Migration commit

mjavint commented:
> Also, do you mind squashing the commit [92764e2](https://github.com/OCA/automation/commit/92764e287b9d7018244d05a1cd4557fbf33d5c4b) into [7d31035](https://github.com/OCA/automation/commit/7d31035b98d6e6d0f94a93d5b9afc4666cc56016)? That way will be much easier to review the changes done during the migration, right now there are a lot of HEAD tags on the Migration commit

Solved in [4b281a0](https://github.com/OCA/automation/pull/16/commits/4b281a0b5343e6284f966460aceb8d33c71e7c67)

arielbarreiros96 reviewed (commented):
Found functional issues when generating the records (see image). I wasn't able to generate records after for example duplicating a configuration (but it also happens if manually created)
![image](https://github.com/user-attachments/assets/aae3036d-a9ef-4f51-9552-6f0af9404a22)

mjavint commented:
> Found functional issues when generating the records (see image). I wasn't able to generate records after for example duplicating a configuration (but it also happens if manually created)
> ![image](https://github.com/user-attachments/assets/aae3036d-a9ef-4f51-9552-6f0af9404a22)
> 

I'm already working on solving it

arielbarreiros96 committed:
[FIX] automation_oca error on record delete

Commit:
[UPD] Update automation_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/

arielbarreiros96 commented:
Hello @mjavint can you consider to incorporate changes made on [this](https://github.com/OCA/automation/pull/15) PR to the migration? Thanks

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (224 of 224 strings)

Translation: automation-16.0/automation-16.0-automation_oca
Translate-URL: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca/it/

christian-ramos-tecnativa commented:
@mjavint can you check the tests? It seems like there is an error

Yiidiir commented:
Thanks for working on this, any updates? @mjavint @Christian-RB

etobella commented:
@mjavint The idea was to have a new icon that is different from Odoos one. Please, don't change it.

antoniodavid committed:
[IMP] automation_oca: pre-commit execution

antoniodavid commented:
@Christian-RB @arielbarreiros96 @etobella I'm resuming this migration started by Manuel.

antoniodavid committed:
[MIG] automation_oca: Migration to 17.0

antoniodavid committed:
[MIG] automation_oca: Migration 17.0

pedrobaeza commented:
Supersede by #28
